exec-service-wasmer/src/wasmer_metering.rs

```diff
@@ -28,18 +28,24 @@ impl MeteringGlobalIndexes {
 }
 
 #[derive(Debug)]
-pub struct Metering {
+pub(crate) struct Metering {
     points_limit: u64,
-    opcode_cost: Rc<OpcodeCost>,
-    global_indexes: Rc<RefCell<Option<MeteringGlobalIndexes>>>,
+    opcode_cost: Arc<OpcodeCost>,
+    breakpoint: Arc<Breakpoint>,
```

Comment:
Please rename the `breakpoint` field to `breakpoints_middleware`.

Reply:
done